client/flutter/lib/components/question_data.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/cupertino.dart';
+
+/// This is just a *placeholder* for the dynamic content.
+class QuestionData {
+  static Future<List<QuestionItem>> questions() async {
+    return [
+      QuestionItem(
+          title:
+          "What are coronaviruses, <b>what is COVID-19</b>, and how is it related to SARS?",
+          body: """
+              <h2>What are the symptoms of COVID-19?</h2>
```

Comment:
How would localization work for this? Would we need to localize this whole chunk of html, or is there a way to localize individual phrases within this single string?

Reply:
By splitting it per section (subtitle, list heading, symptom fever, symptom cough, etc) the chances of breaking HTML and line-break structure lower - but it does increase complexity with more keys and less context.

I know some translation software gives warnings so these are maintained. It's not that clear from Lingo24's website that it does that.

Perhaps @clementmouchet and @sapte91 can advise?

Reply:
Good question but we haven't decided yet... working on it today.

Reply:
Attempting to tag @dleuck

Reply:
The present structure is suitable and preserves context, allowing translators to work on the complete unit. The CAT tool will take care of filtering the HTML, so there's no risk of breaking it, and segmentation will kick in for each bullet point (and sentences inside if any) so that's not an issue at all. This component doesn't have access to the context as far as I can tell on master, so I can't externalise the strings at the moment.
I'm free to work on that with you today @patniemeyer if you can.